util/src/main/java/io/kubernetes/client/informer/ResyncRunnable.java

```diff
@@ -18,30 +18,26 @@
 import org.slf4j.LoggerFactory;
 
 /**
- * ResyncRunnable class implements Runnable interface. It calls the resync function of Store
```

Comment:
there is no sense in saying that `Store` is getting called, when in reality `Store` is not even passed in the constructor. I guess this is an old left over, when the code moved, but the documentation was left intact